resources/lib/metadata.py

```diff
@@ -0,0 +1,584 @@
+                if (offtime - now).days > 0:
+                    plot_meta += self._kodi.localize(30203, days=(offtime - now).days)
+                else:
+                    plot_meta += self._kodi.localize(30204, hours=int((offtime - now).seconds / 3600))
+
+            plot = '%s\n%s' % (plot_meta, plot)
+
+            permalink = statichelper.shorten_link(api_data.get('permalink')) or api_data.get('externalPermalink')
+            if self._showpermalink and permalink:
+                plot = '%s\n\n[COLOR yellow]%s[/COLOR]' % (plot, permalink)
+            return plot
+
+        # VRT NU Suggest API
+        if api_data.get('type') == 'program':
+            plot = statichelper.unescape(api_data.get('description', '???'))
+            permalink = statichelper.shorten_link(api_data.get('targetUrl'))
+            if self._showpermalink and permalink:
+                plot = '%s\n\n[COLOR yellow]%s[/COLOR]' % (plot, permalink)
+            return plot
+
+        # VRT NU Schedule API
+        if api_data.get('vrt.whatson-id'):
+            title = '%s - %s' % (api_data.get('start'), api_data.get('title'))
+            now = datetime.now(dateutil.tz.tzlocal())
+            epg = self.parse(date, now)
+            datelong = self._kodi.localize_datelong(epg)
+            start = api_data.get('start')
+            end = api_data.get('end')
+            plot = '[B]%s[/B]\n%s\n%s - %s\n[I]%s[/I]' % (title, datelong, start, end, channel.get('label'))
+            return plot
+
+        # Not Found
+        return ''
+
+    def get_plotoutline(self, api_data, season=False):
+        ''' Get plotoutline string from single item json api data '''
+        # VRT NU Search API
+        if api_data.get('type') == 'episode':
+            if season:
+                plotoutline = statichelper.convert_html_to_kodilabel(api_data.get('programDescription'))
+                return plotoutline
+
+            if api_data.get('displayOptions', dict()).get('showShortDescription'):
+                short_description = statichelper.convert_html_to_kodilabel(api_data.get('shortDescription'))
+                plotoutline = short_description
+            else:
+                plotoutline = api_data.get('subtitle')
+            return plotoutline
+
+        # VRT NU Suggest API
+        if api_data.get('type') == 'program':
+            return ''
+
+        # VRT NU Schedule API
+        if api_data.get('vrt.whatson-id'):
+            return ''
+
+        # Not Found
+        return ''
+
+    def get_season(self, api_data):
+        ''' Get season int from single item json api data '''
+
+        # VRT NU Search API
+        if api_data.get('type') == 'episode':
+            try:
+                season = int(api_data.get('seasonTitle'))
+            except ValueError:
+                try:
+                    season = int(api_data.get('seasonName'))
+                except ValueError:
+                    season = int()
```

Comment:
This is a weird fallback value. Can you explain ?

Reply:
I return an empty int because the `season` property  for the  `xbmcgui.ListItem` `infoLabels` should be an int. Other return values like `None` or` ''` are also possible though.

https://romanvm.github.io/Kodistubs/_autosummary/xbmcgui.html?highlight=setinfo#xbmcgui.ListItem.setInfo